enhancements/autoscaling/provisioning-request-enablement.md

```diff
@@ -0,0 +1,309 @@
+  and begins to create the requested cluster capacity.
+
+### API Extensions
+
+- Adds the Kubernetes custom resource definition for `provisioningrequests.autoscaling.x-k8s.io`
+  to all OpenShift installations where the Cluster Autoscaler Operator is also installed.
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+Hypershift management clusters utilize the Cluster Autoscaler but do not deploy it using
+the Cluster Autoscaler Operator. As such, this API change will not be deployed in Hypershift
+management clusters unless the Cluster Autoscaler Operator is installed on that cluster.
+In those cases where the management cluster does have the ProvisioningRequest resource, the
+namespaced Cluster Autoscalers deployed by Hypershift do not have the flag enabled for
+reconciliation which eliminates the risk of errant resources causes unwanted scaling.
+
+Hypershift guest clusters do not contain a machine management layer for users and do
+not deploy the Cluster Autoscaler Operator. This change will have no effect on those clusters.
+
+#### Standalone Clusters
+
+This change is directly affecting standalone clusters, it does the following:
+
+* Adds the `provisioningrequests.autoscaling.x-k8s.io` resource type to the API server.
+* Configures the Cluster Autoscaler to use the `--enable-provisioning-requests` flag.
+* Updates the Cluster Autoscaler Operator RBAC to allow modification of ProvisioningRequests.
+
+#### Single-node Deployments or MicroShift
+
+This change does not affect single-node deployments and Microshift as they do not
+install the Cluster Autoscaler Operator.
+
+### Implementation Details/Notes/Constraints
+
+The main implementation detail for this enhancement is the inclusion of the
+ProvisioningRequest custom resource definition.
+
+Currently, the Cluster Autoscaler Operator contains custom resource definitions for
+the ClusterAutoscaler and MachineAutoscaler types in its payload manifests. Following
+in this pattern, the `Makefile` for the Cluster Autoscaler Operator will be modified to
+include a target for rendering the ProvisioingRequest manifest from vendored files
+containing the Cluster Autoscaler API package. The resulting custom resource definition
+manifest will be included in the payload manifests for the operator.
+
+The second implementation detail worth noting is the change in default configuration
+for the Cluster Autoscaler. In order to properly reconcile ProvisioingRequests in
+a cluster, the Cluster Autoscaler must be started with the command line flag
+`--enable-provisioning-requests=true`. The Cluster Autoscaler Operator will be
+updated to include this flag as a default. There will not be a corresponding
+API field to allow user configuration of this flag.
+
+### Risks and Mitigations
+
+This change is generally low risk but there are a few concerns to consider:
+
+1. Unauthorized creation of infrastructure resources. Because the ProvisioningRequest
```

Comment:
is this referring to standard kube RBAC or is there other concerns?

Reply:
i updated the text to mention the kube rbac and rationale.